101-notification-hub/azuredeploy.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "namespaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Notification Hubs namespace."
+            }
+        },
+        "hubName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Notification Hubs hub."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location in which the Notification Hubs resources should be deployed."
+            }
+        }
+    },
+    "variables": {
+        "notificationHubsApiVersion": "2014-09-01"
+    },
+    "resources": [
+        {
+            "apiVersion": "[variables('notificationHubsApiVersion')]",
```

Comment:
use a literal apiVersion - otherwise json editors can use the schema for intellisense